namespace-cli-todo.md

```diff
@@ -0,0 +1,32 @@
+# Namespace CLI
+
+Make a single cli command for namespaces contract, testing out namespace creation. This will be similar to the existing command for item.
+
+## TODO
+
+Some things left to do or updates to make to work with latest anchor/package versions
+
+## Anchor updates
+
+* Anchor >0.20.1 now does not support using bump on init. <https://github.com/project-serum/anchor/commit/598c7b0790689187360ffbf80935dac94d9a939c>
```

Comment:
Yeah in anchor 0.20.1+ the bumps get stored after the discriminator. I'm not sure how to migrate off the top of my head.